docs/docs/dev/security/certificates.md

```diff
@@ -122,6 +122,46 @@ You can also provide the CA via environment variable `KUMA_CONTROL_PLANE_CA_CERT
 
 See [Data plane proxy authentication](./dp-auth.md) and [Zone Ingress authentication](./zone-ingress-auth.md).
 
+## Prometheus to control plane communication
+
+It is possible to enabled TLS on the Monitoring Assignment Discovery Service.
+By default it will use the certificate used in `tls-general` but TLS will be disabled.
+You can enable it by using either the environment variable `KUMA_MONITORING_ASSIGNMENT_SERVER_TLS_ENABLED=true` or the yaml config `monitoringAssignmentServer.tlsEnabled=true`.
+
+:::: tabs :options="{ useUrlFragment: false }"
+::: tab "Kubernetes (kumactl)"
+Create a secret in the namespace where the control plane is installed:
+```sh
+kubectl create secret generic general-tls-certs -n <namespace> \
+  --from-file=tls.crt=/tmp/tls.crt \
+  --from-file=tls.key=/tmp/tls.key \
+  --from-file=ca.crt=/tmp/ca.crt
+```
+
+Point to this secret when installing Kuma:
+```sh
+kumactl install control-plane \
+  --tls-general-secret=general-tls-certs \
+  --tls-general-ca-bundle=$(cat /tmp/ca.crt | base64) \
+  --env-var 'KUMA_MONITORING_ASSIGNMENT_SERVER_TLS_ENABLED=true'
+```
+
+:::
+::: tab "Universal"
+Configure the control plane with generated certificates:
+
+```sh
+KUMA_GENERAL_TLS_CERT_FILE=/tmp/tls.crt \
+  KUMA_GENERAL_TLS_KEY_FILE=/tmp/tls.key \
+  KUMA_MONITORING_ASSIGNMENT_SERVER_TLS_ENABLED=true \
+  kuma-cp run
+```
+
+:::
+::::
+
+You can then configure Kuma's prometheus SD with the right TLS configuration using the [Prometheus docs](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#kuma_sd_config).
```

Comment:
```suggestion
Now you can configure Kuma's Prometheus SD with the correct TLS configuration using the [Prometheus docs](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#kuma_sd_config).
```
I'm not sure what "SD" stands for here, but if it's an acronym I'd spell it out first like "Prometheus Service Discovery (SD)".